db/migrate/20220430074045_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author_id, null: false, foreign_key: { to_table: :users }
+      t.references :comment_id, null: false, foreign_key: { to_table: :posts }
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Please, consider that you have to add the `text` field also, considering the ERD (hint: `t.text :text`)
<img width="217" alt="Screen Shot 2022-04-30 at 12 23 08" src="https://user-images.githubusercontent.com/49064106/166098092-3e81b951-0f55-4de8-a63a-1f422b0b2bea.png">